app/mailers/patient_mailer.rb

```diff
@@ -14,6 +14,7 @@ def enrollment_email(patient)
       { patient: dependent, jurisdiction_unique_id: Jurisdiction.find_by_id(dependent.jurisdiction_id).unique_identifier }
     end
     @lang = patient.select_language
+    @contact_info = Jurisdiction.find_by_id(patient.jurisdiction_id).contact_info
```

Comment:
You can simplify this by doing:
```
@contact_info = patient.jurisdiction.contact_info
```

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/925/commits/860024d23a6f4b689c1baa9bc9cd315a5e456796